### DISCUSSION TITLE
Free Monitor - Service Notifications from another physical devices

### DISCUSSION THREAD

**Discussion by paulzhdanov** [RavenSystem/esp-homekit-devices#1720] [Q&A]
@RavenSystem could you please clarify: will it be possible to get Service Notifications from another physical devices? Within Free Monitor concept it would give fantastic opportunities and flexible information exchange, for instance, distributed thermostats where sensor and action device are in different rooms

**Comment by RavenSystem** [comment]
I don't understand.

**Reply by paulzhdanov** [reply]
Please excuse my poor English. Let's assume we have 2 different device flashed with RavenSystem firmware. Is there any chance to send notifications from one to another?

**Reply by RavenSystem** [reply]
You can use network request actions to send data or orders to other device. In other way, HAA can read data of other devices using Free Monitor with network request type.  If target device is HAA, you must enable unsecure mode on it.

**Reply by paulzhdanov** [reply]
Could you please show any example of how this could be done? In ideal HeaterCooler (t:25) with remote temperature sensor?
